### ISSUE TITLE
[tune] display any field in results.json from tune cli

### ISSUE THREAD

**Issue by BenMacKenzie** [ray-project/ray#4531]
i'd like to do something like this:

tune list-experiments ~/ray_results --sort accuracy 

i.e., sort by and display any field in the results.json file.  maybe i am missing something, but it is not clear how else to see results from each experiment, apart from watching the output from run_experiments or searching manually.

**Comment by richardliaw** [comment]
Hey @BenMacKenzie, this is blocked by #4519. I think this issue is a duplicate of #4523; can you take a look there to see if it is? (Feel free to leave some thoughts there and close the issue if it is duplicate).

Thanks!

**Comment by BenMacKenzie** [comment]
yeah...looks like it.  I'll close it.
